packages/server/graphql/mutations/helpers/notifySlack.ts

```diff
@@ -111,15 +161,36 @@ export const endSlackMeeting = async (meetingId, teamId, dataLoader: DataLoaderW
     utm_campaign: 'after-meeting'
   }
   const options = {searchParams}
-  const team = await dataLoader.get('teams').load(teamId)
+  const [team, meeting] = await Promise.all([
+    dataLoader.get('teams').load(teamId),
+    dataLoader.get('newMeetings').load(meetingId)
+  ])
   const summaryUrl = makeAppURL(appOrigin, `new-summary/${meetingId}`, options)
-  const slackText = `The meeting for ${team.name} has ended!\n Check out the summary here: ${summaryUrl}`
-  notifySlack('meetingEnd', dataLoader, teamId, slackText).catch(console.log)
+  const meetingUrl = makeAppURL(appOrigin, `meet/${meetingId}/discuss/1`)
```

Comment:
discuss only exists on retros, what about check-ins?

i see a lot of conditional logic based on meeting type. might be worthwhile branching this logic based on meetingType.

e.g. `const blocks = makeBlocksForMeeting()`
makeBlocksForMeeting = () => {
  switch (meeting.meetingType) {
    case 'retro': 
    case 'action':
      return foo
  }
}
```

Reply:
True, it looks like most of the conditional logic is in the end meeting notification. I've created a couple of functions to make it a bit more readable